guides/common/modules/proc_deploying-a-custom-ssl-certificate-to-hosts.adoc

```diff
@@ -1,14 +1,58 @@
 [id="deploying-a-custom-ssl-certificate-to-hosts_{context}"]
-
 = Deploying a Custom SSL Certificate to Hosts
 
-After you configure {ProductName} to use a custom SSL certificate, you must also install the `katello-ca-consumer` package on every host that is registered to this {ProductName}.
+After you configure {Project} to use a custom SSL certificate, you must deploy the certificate to hosts registered to {Project}.
 
 .Procedure
-
-* On each host, install the `katello-ca-consumer` package:
+* Update the SSL certificate on each host:
 +
+ifdef::satellite[]
 [options="nowrap", subs="+quotes,attributes"]
 ----
 # {client-package-install-el8} http://_{context}.example.com_/pub/katello-ca-consumer-latest.noarch.rpm
 ----
+endif::[]
+ifndef::satellite,orcharhino[]
+** On Debian and Ubuntu:
++
+[options="nowrap" subs="+quotes,attributes"]
+----
+# wget http://_{context}.example.com_/pub/katello-rhsm-consumer
+# chmod +x katello-rhsm-consumer
+# ./katello-rhsm-consumer
+----
+** On {EL} 8+:
++
+[options="nowrap" subs="+quotes,attributes"]
+----
+# {client-package-install-el8} http://_{context}.example.com_/pub/katello-ca-consumer-latest.noarch.rpm
+----
+** On {EL} 7:
++
+[options="nowrap" subs="+quotes,attributes"]
+----
+# {client-package-install-el7} http://_{context}.example.com_/pub/katello-ca-consumer-latest.noarch.rpm
+----
+** On OpenSUSE and {SLES}:
++
+[options="nowrap" subs="+quotes,attributes"]
+----
+# {client-package-install-sles} http://_{context}.example.com_/pub/katello-ca-consumer-latest.noarch.rpm
+----
+endif::[]
+ifdef::orcharhino[]
+ifdef::debian,ubuntu[]
+[options="nowrap", subs="+quotes,attributes"]
+----
+# wget http://_{context}.example.com_/pub/katello-rhsm-consumer
+# chmod +x katello-rhsm-consumer
+# ./katello-rhsm-consumer
+----
+endif::[]
+ifndef::debian,ubuntu[]
+[options="nowrap", subs="+quotes,attributes"]
+----
+# {client-package-install} http://_{context}.example.com_/pub/katello-ca-consumer-latest.noarch.rpm
+----
+endif::[]
```

Comment:
Did you intend to have nested conditions? (2 ifdefs on lines 43-44, 2 endifs on lines 57-58). If so, ignore this comment.

Reply:
Yes, this is intentional.